### DISCUSSION TITLE
Mode : Join TCP/IP Game

### DISCUSSION THREAD

**Discussion by ifeelingz** [isid0re/SoloLeveling#374] [Q&A]
Mode : Join TCP/IP Game is stuck on the IP Address page if I want to specify IP or press OK, where do I have to modify it ?

![image](https://user-images.githubusercontent.com/6375304/136305171-ac8d690b-f974-4a2a-93ad-4d0b14b3f03c.png)

Thank you ☺

**Comment by isid0re** [comment]
What entry are you trying to use when that pop up appears?

**Reply by isid0re** [reply]
Bot isnt set up for joining tcpip games. You should select ` Host: tcpip game` and that fix your problems.

**Reply by ifeelingz** [reply]
Thanks.

**Comment by jh86** [comment]
https://github.com/kolton/d2bot-with-kolbot/issues/1496

This is a workaround - currently using it. I added a little bit more instruction at the bottom of the post to ensure you get it working!
